src/pf-tabs/index.js

```diff
@@ -0,0 +1,5 @@
+/** PF Tabs Component **/
+require('./pf-tab.component');
+require('./pf-tab.template');
+require('./pf-tabs.component');
+require('./pf-tabs.template');
```

Comment:
I don't understand why this is required considering tab and tabs import their own dependencies?